akka-http2-support/src/main/scala/akka/http/impl/engine/http2/Http2Blueprint.scala

```diff
@@ -82,9 +85,9 @@ object Http2Blueprint {
     }
     BidiFlow.fromFlows(
       Flow[HttpResponse].map(ResponseRendering.renderResponse(settings, log)),
-      Flow[Http2SubStream].via(StreamUtils.statefulMap { () ⇒
+      Flow[Http2SubStream].via(StreamUtils.statefulAttrsMap { attrs ⇒
```

Comment:
It's even needed to run this function only once, as explained above, otherwise, the parser will be recreated every time and we will use the internal caching of the parser.